azurerm/internal/services/compute/tests/resource_arm_dedicated_host_group_test.go

```diff
@@ -0,0 +1,180 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/features"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMDedicatedHostGroup_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_dedicated_host_group", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMDedicatedHostGroupDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMDedicatedHostGroup_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMDedicatedHostGroupExists(data.ResourceName),
+				),
+			},
+			data.ImportStep(),
+		},
+	})
+}
+
+func TestAccAzureRMDedicatedHostGroup_requiresImport(t *testing.T) {
+	if !features.ShouldResourcesBeImported() {
+		t.Skip("Skipping since resources aren't required to be imported")
+		return
+	}
+
+	data := acceptance.BuildTestData(t, "azurerm_dedicated_host_group", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMDedicatedHostGroupDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMDedicatedHostGroup_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMDedicatedHostGroupExists(data.ResourceName),
+				),
+			},
+			data.RequiresImportErrorStep(func(data acceptance.TestData) string {
```

Comment:
since they have the same method signature we can simplify this to:

```suggestion
			data.RequiresImportErrorStep(testAccAzureRMDedicatedHostGroup_requiresImport),
```